### PR TITLE
Created basic options page

### PR DESCRIPTION
**Pull request by kabirgh**
The page can be viewed at `chrome://extensions` => Spoiler Blocker => Options.

### CODE REVIEW

**Review by varung97 — CHANGES_REQUESTED** [review]
I think the default preferences aren't getting saved to the storage. I tried changing default case sensitivity to true, but when I opened the options page again, it was back to false.

File: src/options/options.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+				{"New lists created will have the following default preferences."}
+				<br /><br/>
+
+				<Switch
+					label="Case sensitive"
+					checked={OptionStore.defaultCaseSensitivity}
+					onChange={this.handleToggleCaseSensitivity}
+				/>
+				<Switch
+					label="Overlay/remove" // TODO: clearer label
```
Comment (varung97) [inline src/options/options.js:44]: Should we change the label to `Overlay/Remove`?

Comment (kabirgh) [reply src/options/options.js:44]: I wanted to change this to something more descriptive, actually. Just copied this short form from the options menu of the card as a placeholder.

**Comment by kabirgh** [comment]
The new commits should fix the problem. I had forgotten to sync the prefs to chrome storage haha.

**Comment by varung97** [comment]
Nice, works perfectly 👍

**Pull request merged**